packages/core/src/components/tag/tag.tsx

```diff
@@ -165,13 +165,7 @@ export class Tag extends AbstractPureComponent2<TagProps> {
         );
         const isLarge = large || tagClasses.indexOf(Classes.LARGE) >= 0;
         const removeButton = isRemovable ? (
-            <button
-                aria-label="Remove Tag"
-                type="button"
-                className={Classes.TAG_REMOVE}
-                onClick={this.onRemoveClick}
-                tabIndex={interactive ? tabIndex : undefined}
-            >
+            <button aria-label="Remove Tag" type="button" className={Classes.TAG_REMOVE} onClick={this.onRemoveClick}>
```

Comment:
shouldn't this have `tabIndex={tabIndex}`?

Reply:
that works, changed